apache-maven/src/assembly/shared/mvnvalidate.ps1

```diff
@@ -0,0 +1,7 @@
+# check mvn home
+$MAVEN_HOME = (Get-Item $PSScriptRoot).Parent
+
+# check if maven command exists
+if (-not (Test-path "$MAVEN_HOME\bin\mvn.ps1")) {
+    Write-Error -Message "maven command (\bin\mvn.ps1) cannot be found" -ErrorAction Stop
```

Comment:
Why not `($MAVEN_HOME \bin\mvn.ps1)`?
Do we have such a message with the other scripts?

Reply:
No, we don't have such a message with the other scripts. From my point of view if the maven command can't be found, the script must stop and print an error so the problem can be resolved. 

Although this would be an edge case.

Reply:
Should be identical to others